docs/CHANGELOG.rst

```diff
@@ -19,7 +19,11 @@ Changed
 Added
 -----
 - Support Python 3.7
+- Add ``bamclipper`` tool and ``parallel`` package to ``resolwebio/dnaseq:4.2.0`` image
```

Comment:
Add `bamclipper` to the readme file of the Docker image as well.

Reply:
Done.